bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/messages.properties

```diff
@@ -568,6 +568,23 @@ CheckedTreeSelectionDialog_nothing_available=No entries available.
 CheckedTreeSelectionDialog_select_all=Select &All
 CheckedTreeSelectionDialog_deselect_all=&Deselect All
 
+WindowsDefenderConfigurator_statusCheck=Windows Defender Exclusion Check
+WindowsDefenderConfigurator_exclusionCheckMessage=Microsoft's Windows Defender is active and could significantly decrease the application's start-up and overall performance.\n\
+Select how this installation should be handled by Windows Defender:
+WindowsDefenderConfigurator_exclusionInformation=If Windows Defender is active and scans this application it can significantly slow down its start-up and overall performance.\n\
```

Comment:
```suggestion
WindowsDefenderConfigurator_exclusionInformation=If Windows Defender is active and scans this application it can significantly slow down it's start-up and overall performance.\n\
```

Reply:
In `If Windows Defender is active and scans this application it can significantly slow down its start-up ...` the `its` should be the genitive of `it` (`the start-up of the application`) and and not the short form of `it is`.
I'm not the best English speaker/writer, but I think this is right as it is.

Reply:
Sure. Your are right.